conf/ledgersmb-nginx.template

```diff
@@ -0,0 +1,63 @@
+##
+# LedgerSMB server configuration
+#
+server {
+        listen 80 default_server;
+        listen [::]:80 default_server;
+
+        #Set to LSMB installed directory
+        root [ROOT];
+
+        index login.pl;
+
+        #Adjust to your liking
+        #server_name ubuntu-vm;
+
+	#Try the asked file directly, pass to Plack if not found
+        location / {
```

Comment:
Shouldn't we a bit more strict? We already know where the static files are and what their extensions are:
* css/
* image/*
* UI/**/*.js